infrastructure/src/main/java/org/wso2/testgrid/infrastructure/providers/ShellScriptProvider.java

```diff
@@ -45,21 +45,16 @@ public String getProviderName() {
 
     @Override
     public boolean canHandle(Infrastructure infrastructure) {
-        boolean isScriptsAvailable = true;
-        for (Script script : infrastructure.getScripts()) {
-            if (!Script.ScriptType.INFRA_CREATE.equals(script.getScriptType()) &&
-                !Script.ScriptType.INFRA_DESTROY.equals(script.getScriptType())) {
-                isScriptsAvailable = false;
-            }
+        if (infrastructure.getProviderType().equals(Infrastructure.ProviderType.SHELL)) {
+            return true;
         }
-        return isScriptsAvailable;
+        return false;
```

Comment:
We can simply return `infrastructure.getProviderType().equals(Infrastructure.ProviderType.SHELL)` :)

Reply:
This is fixed.